src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int maxNumber = 100;
```

Comment:
also nextInt() returns int from 0 to `<` passed variable, not 0 `<= `

Reply:
fixed